bundles/org.openhab.binding.fenecon/README.md

```diff
@@ -0,0 +1,146 @@
+# FENECON Binding
+
+The FENECON Binding integrates the [FENECON energy storage system](https://fenecon.de/) device into the openHAB system via [REST-API](https://docs.fenecon.de/_/de/fems/fems-app/OEM_App_REST_JSON.html).
+
+With the binding, it is possible to request status information from FENECON Home to allow you home automation decisions based on the current energy management.
+
+This makes it possible, for example, to switch on other consumers such as the dishwasher or washing machine in the case of power overproduction.
+
+## Supported Things
+
+Only one Thing is supported: The `fenecon` device.
+
+This Binding was tested with an [FENECON HOME 10](https://fenecon.de/fenecon-home-10/) device.
+
+## Discovery
+
+Auto-discovery is not supported.
+
+## Thing Configuration
+
+The FENECON Thing only needs to be configured with the `hostname`, all other parameters are optional and prefilled with the suitable default values:
+
+| Parameter       | Description                                                                      |
+|-----------------|----------------------------------------------------------------------------------|
+| hostname        | Hostname or IP address of the FENECON device, e.g. 192.168.1.11                  |
+| password        | Password of the FENECON device. The password for guest access is set by default. |
+| port            | Port of the FENECON device. Default: 8084                                        |
+| refreshInterval | Interval the device is polled in sec. Default 30 seconds                         |
+
+
+## Channels
+
+The FENECON binding currently only provides access to read out the values from the energy storage system. 
+
+| Channel                | Type                 | Read/Write | Description                                                                 |
+|------------------------|----------------------|------------|-----------------------------------------------------------------------------|
+| state                  | String               | R          | FENECON system state: Ok, Info, Warning or Fault                            |
+| lastUpdate             | DateTime             | R          | Last successful update via REST-API from the FENECON system                 |
```

Comment:
Please adjust all channel id's according to the naming convention `lower-case-hyphen`. Readme, thing-types.xml, binding constants and any other place these are used.
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Reply:
Done